package.json

```diff
@@ -24,6 +24,7 @@
   "homepage": "https://github.com/labritz/roomier#readme",
   "dependencies": {
     "@react-google-maps/api": "^2.12.1",
+    "axios": "^1.2.2",
```

Comment:
Do you need Axios? Can you use node-fetch since the package is already integrated? Otherwise we are porting over two dependencies that will do the same thing